user_modules/shared_testcases/testCasesForPolicyAppIdManagament.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------
+-- Policy: AppID Management common module
+---------------------------------------------------------------------------------------------
+
+local common = {}
+
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+local policy_file_path = commonFunctions:read_parameter_from_ini("SystemFilesPath")
```

Comment:
@istoimenova Ok we discussed with Kovalenko I. and he said that your way is better than my.